<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wRole</class>
 <widget class="QWidget" name="wRole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Łachut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="topMargin">
      <number>80</number>
     </property>
     <property name="rightMargin">
      <number>150</number>
     </property>
     <property name="bottomMargin">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="hosterBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Twórca quizu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="userBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Uczestnik quizu</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
